just-install-v4.json

```diff
@@ -2572,6 +2572,25 @@
       },
       "version": "latest"
     },
+    "visual-studio-vctools-2019": {
```

Comment:
I'm not convinced this package name is the best idea. The other `visual-studio-*` packages refer to visual studio editions, not components.

I think `visual-studio-2019-vctools` or `vctools-2019` would be clearer. @lvillani?

Reply:
I don't know. Microsoft uses so many slightly different names to refer to this workload. I'm inclined to merge this as-is for now and rename them at a later date if we come up with a better naming scheme.